boards/PDM/Inc/App/configs/App_SharedSignalConfig.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+struct PdmWorld;
```

Comment:
Had to add this for all boards as required by `SharedSignal.h` to compile successfully (Even if a board isn't using `Signal`)